syntax = "proto3";

package image_service;

message ImageRequest {
    bytes image_data = 1;
}

message ImageConfirmation {
    bool success = 1;
    string message = 2;
}

service ImageService {
    rpc SendImage(ImageRequest) returns (ImageConfirmation);
}